docs/one-click-integrations.md

```diff
@@ -22,15 +36,25 @@ Pulse integrates directly with GitHub to receive data about changes and deployme
 
         Because of this, the integration discards the first SemVer tag in the repository since there is no previous tag to compare with.
 
-To set up the GitHub integration:
+    -   Make sure that you're creating a Git tag on your repositories for each successful deployment to production, or whenever you make a new release available to any user of your application:
+
+            ```bash
+            git tag -a X.Y.Z -m "<Deployment or release message>"
+            ```
 
-1.  Make sure that you're creating a Git tag on your repositories for each successful deployment to production, or whenever you make a new release available to any user of your application:
+            Where X.Y.Z must be a valid SemVer version without pre-release information.
 
-    ```bash
-    git tag -a X.Y.Z -m "<Deployment or release message>"
-    ```
+#### Strategy selection
```

Comment:
Will users be able to configure the deployment detection strategy "on the fly", even if the GitHub App has already been installed?

Either way, my suggestion is that we include the selection of the detection strategy in the "installation flow", both on the documentation and on the UI. It should be either the first or last step in the installation procedure.

Like this, we can simplify a bit and just have a list of instructions that guides the user through installing the GitHub App and deciding what's the best detection strategy for their scenario. This helps to ensure that users don't miss the configuration part of the process. :ok_hand:

Reply:
The UI is not done yet, but the idea was to be able to select the strategy whenever you want.
This part needs to be updated when we finish it.